.github/workflows/controller-test.yml

```
@@ -30,6 +30,14 @@ jobs:
     name: Run on Ubuntu
     runs-on: ubuntu-latest
     steps:
+      - name: Generate tag
+        shell: bash
+        id: tags
+        run: |
+          commit_sha=${{ github.event.after }}
+          tag=main-${commit_sha:0:7}
```

```suggestion
          tag="$BRANCH"-${commit_sha:0:7}
```
Also totally fine as is - this would only matter if we used this for other branches too